msi/rust.wxs

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+    <Product Id="*"
+        Name="Rust $(env.CFG_CHANNEL)"
+        Language="1033"
+        Version="$(env.CFG_MSI_VERSION)"
+        UpgradeCode="$(env.CFG_UPGRADE_CODE)"
+        Manufacturer="Mozilla Foundation">
+        <Package
+            Comments="Rust is a systems programming language that runs blazingly fast, prevents almost all crashes, and eliminates data races."
+            InstallerVersion="200"
+            Compressed="yes" />
+
+        <Icon Id="rust.ico" SourceFile="rust-logo.ico" />
+        <Property Id="ARPPRODUCTICON" Value="rust.ico" />
+        <!-- txt format is not supported -->
+        <WixVariable Id="WixUILicenseRtf" Value="LICENSE.rtf" />
+
+        <!--
+            Source media for the installation.
+            Specifies a single cab file to be embedded in the installer's .msi.
+        -->
+        <Media Id="1" Cabinet="contents.cab" EmbedCab="yes" CompressionLevel="high" />
```

Comment:
@Arke64 says: "`<MediaTemplate EmbedCab="yes" />` is a more concise way than [that]".